polars/dataframe.ts

```diff
@@ -1755,6 +1801,66 @@ export const _DataFrame = (_df: any): DataFrame => {
     set columns(names) {
       _df.columns = names;
     },
+    /**
+     * Return back text/html and application/vnd.dataresource+json representations of the DataFrame
+     * @returns Media bundle / mimetype keys for Jupyter frontends
+     */
+    [Symbol.for("Jupyter.display")]() {
+      // Implementation of the Tabular Data Resource Schema. Also used for text/html representation
+      type DataResourceField = {
+        name: string;
+        type: string;
+      };
+
+      const fields = this.getColumns().reduce(
+        (acc: DataResourceField[], curr) => {
+          acc.push({
+            name: curr.name,
+            type: mapPolarsTypeToJSONSchema(curr.dtype),
+          });
+          return acc;
+        },
+        [],
+      );
```

Comment:
couldn't you just do a `map` here instead of a `reduce`? 

```js
const fields = this.getColumns().map(column => ({
            name: column.name,
            type: mapPolarsTypeToJSONSchema(column.dtype),
          })
```

Reply:
So true, thank you.